### PR TITLE
feat(curriculum): add cat photo app workshop

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Summary
I kept everything the same from what is currently in main, except for the following:

- updated the description for step 1
- removed the form steps. there is a whole module just for forms and tables

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by jdwilkin4** [comment]
@moT01 

> Changes are still being made to the original Cat Photo App project to fix tests and descriptions, etc. So if we add this now, by the time we want to make it live, the original project will have a bunch of fixes that this doesn't have. 

For this particular project, I vote we merge in this revised version now, and then a few weeks before launch open another PR to add in the updated changes. 
I created an issue to track this

> Wondering if we should wait to add these ones that are duplicates until closer to release. Or maybe we can find a way to not have to duplicate this and just use it in two places.

So essentially there will be 3 types for workshops:

- new workshops
- revised workshops 
- original workshops 

For the new workshops and revised workshops, like this cat photo app, we should open PR's for them when ready so we have plenty of time for reviews. And then if one of these workshops has changes, then we can update a PR to apply those changes before launch.

For the original workshops were we wont' change anything, we can wait to add those closer to the launch.
An example of this would be the Cafe Menu for example

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1700,9 +1700,12 @@
     "title": "Front End Development",
     "intro": ["placeholder"],
     "blocks": {
-      "learn-basic-html-by-building-a-recipe-page": {
-        "title": "Learn Basic HTML by Building a Recipe Page",
-        "intro": ["", ""]
+      "workshop-cat-photo-app": {
+        "title": "Basic HTML Workshop - Build a Cat Photo App",
```
Comment (moT01) [inline client/i18n/locales/english/intro.json:1704]: I don't think we should have this as the title. 

We intend to have something in the UI to show users that this is in the Basic HTML module, and a label for the workshop. So I think we should just go with Build a Cat Photo App, or Learn (something) by Building a Cat Photo App.

File: curriculum/challenges/_meta/workshop-cat-photo-app/meta.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
+  "name": "Basic HTML Workshop - Build a Cat Photo App",
```
Comment (moT01) [inline curriculum/challenges/_meta/workshop-cat-photo-app/meta.json:2]: Same thing about the title here.

File: client/i18n/locales/english/intro.json (hunk 1 above)
Comment (jdwilkin4) [reply client/i18n/locales/english/intro.json:1704]: Ok, Let's go with  Build a Cat Photo App 👍

**Review by moT01 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690b9626a66f51257ae5a91.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+```html
+<h1>most important heading element</h1>
+<h2>second most important heading element</h2>
+<h3>third most important heading element</h3>
+<h4>fourth most important heading element</h4>
+<h5>fifth most important heading element</h5>
+<h6>least important heading element</h6>
+```
+
+Only use one `h1` element per page and place lower importance headings below higher importance headings.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6690b9626a66f51257ae5a91.md:21]:
```suggestion
Only use one `h1` element per page and place lower-importance headings below higher-importance headings.
```

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698419d8f52cf3891f34853.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+---
+id: 6698419d8f52cf3891f34853
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+In previous steps you used an anchor element to turn text into a link. Other types of content can also be turned into a link by wrapping it in anchor tags.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698419d8f52cf3891f34853.md:10]:
```suggestion
In previous steps, you used an anchor element to turn text into a link. Other types of content can also be turned into a link by wrapping it in anchor tags. 
```

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/669842f261665d3b4a42df81.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+---
+id: 669842f261665d3b4a42df81
+title: Step 19
+challengeType: 0
+dashedName: step-19
+---
+
+# --description--
+
+When you add a lower rank heading element to the page, it's implied that you're starting a new subsection.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/669842f261665d3b4a42df81.md:10]:
```suggestion
When you add a lower-rank heading element to the page, it's implied that you're starting a new subsection.
```

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/66984905e95af5432faa28b1.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
+---
+id: 66984905e95af5432faa28b1
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+To improve accessibility of the image you added, add an `alt` attribute with the text:
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/66984905e95af5432faa28b1.md:10]:
```suggestion
To improve the accessibility of the image you added, add an `alt` attribute with the text:
```

File: curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698510677b5e655ba8c1017.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+# --description--
+
+You can set browser behavior by adding `meta` elements in the `head`. Here's an example:
+
+```html
+<meta attribute="value">
+```
+
+Inside the `head` element, nest a `meta` element with an attribute named `charset`. Set to the value to `utf-8` which tells the browser how to encode characters for the page.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698510677b5e655ba8c1017.md:16]:
```suggestion
Inside the `head` element, nest a `meta` element with a `charset` attribute set to the value of `utf-8`. This tells the browser how to encode characters for the page. 
```

**Review by Ksound22 — COMMENTED** [review]
I only found a few nitpicks.

Other than that, things look good!

**Review by Ksound22 — APPROVED** [review]

**Pull request merged**